src/core/build/types.ts

```diff
@@ -64,7 +64,7 @@ export async function writeTypes(nitro: Nitro) {
         continue;
       }
       let path = resolveAlias(i.from, nitro.options.alias);
-      if (!isAbsolute(path)) {
+      if (isAbsolute(path)) {
```

Comment:
Nuxt implementation has `!isAbsolute(path)`:

https://github.com/nuxt/nuxt/blob/bc669cba0f39d359e183f4eb94d9cbaf1aad3fac/packages/nuxt/src/imports/module.ts#L187-L198

Reply:
I need to edit this in Nuxt, it creates long urls there too and this sometimes causes errors.

Reply:
Nuxt is working correctly because when it comes up it comes in the form of packet names. But in nitro it comes completely as a path, whereas it was a packet.